src/components/ListOfGoods/ListOfGoods.tsx

```diff
@@ -0,0 +1,17 @@
+import './ListOfGoods.scss';
+
+type Props = {
+  goods: string[],
+};
+
+export const ListOfGoods: React.FC<Props> = ({ goods }) => {
+  return (
+    <ul className="list">
+      {
+        goods.map(good => (
+          <li key={good} className="list__item">{good}</li>
```

Comment:
map's callback also accepts the index. you can use that as a stronger version of the key.

Reply:
I thought index can't be used as a key. I get an error when I use it.

Reply:
Yes, you are right. I thought of an id for some reason. But the items in the array are just strings, so they don't have IDs. Never mind the previous comment.